cpp/src/io/comp/nvcomp_adapter.cpp

```diff
@@ -65,8 +65,8 @@
 #endif
 
 // Issue https://github.com/NVIDIA/spark-rapids/issues/6614 impacts nvCOMP 2.4 ZSTD decompression on
-// compute 6.1
-#if NVCOMP_MAJOR_VERSION > 2 or (NVCOMP_MAJOR_VERSION == 2 and NVCOMP_MINOR_VERSION >= 4)
+// compute 6.x
+#if NVCOMP_MAJOR_VERSION == 2 and NVCOMP_MINOR_VERSION == 4
```

Comment:
Don't we need to check the patch version == 0 as well?
Otherwise this will disable it for 2.4.1 as well.